lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
```

Comment:
I presume:
-2nd parameter of security settings is the subroutine implementing a "Security Scheme Object"? (http://swagger.io/specification/#securitySchemeObject)
-1st parameter is the name of the "Security Scheme Object", in this test case it would be 'dummy'?

The naming of the variable $config is obscuring. It should explicitly state that it is a anonymous subroutine, even better, of the type "Security Scheme Object"-implementation.
Maybe something like
    $securitySchemeObjectImplementation